disposable_email_blacklist.conf

```diff
@@ -2054,6 +2108,8 @@ spambox.info
 spambox.irishspringrealty.com
 spambox.org
 spambox.us
+spamcannon.com
+spamcannon.net
```

Comment:
Maybe this one should go to `whitelist` too, ping @martenson

Reply:
Sorry @martenson ignore this comment as `spamcannon.net` is already in the whitelist